src/main/java/com/yelp/nrtsearch/server/luceneserver/field/LatLonFieldDef.java

```diff
@@ -122,4 +125,14 @@ public SortField getSortField(SortType type) {
     return LatLonDocValuesField.newDistanceSort(
         getName(), origin.getLatitude(), origin.getLongitude());
   }
+
+  @Override
+  public Query getGeoBoundingBoxQuery(GeoBoundingBoxQuery geoBoundingBoxQuery) {
```

Comment:
nit: can this throw an exception if the field is not searchable.

Reply:
Done!